src/utils/priceFormat.ts

```diff
@@ -0,0 +1,16 @@
+
+const priceFormat = (tilePrice: number, deliveryPrice: number, installPrice: number ) => {
+  let totalPrice = (tilePrice + deliveryPrice + installPrice).toFixed(2);
+  let decimal = totalPrice.indexOf('.');
+  if (decimal < 0) { 
+    decimal = totalPrice.length; 
+    totalPrice += '.'; 
+  }
+  while (totalPrice.length <= decimal + 2) { 
+    totalPrice += '0'; 
+  } 

```

Comment:
I'm confused about the purpose of these codes, is it for making "1234" to "1234.00"? If so, I think the .toFixed(2) already did that for you.

P.S. single-quoted, prettier did not work?